apps/src/applab/commands.js

```diff
@@ -192,6 +204,7 @@ applabCommands.button = function (opts) {
 };
 
 applabCommands.image = function (opts) {
+  apiValidateDomIdExistence(opts, 'image', 'id', opts.elementId, false);
```

Comment:
Good catch. It looks like the following commands also need to call `apiValidateDomIdExistence`:
```
applabCommands.container
applabCommands.imageUploadButton
```